crates/symbolicator/src/services/symbolication/comparisons.rs

```diff
@@ -111,9 +111,8 @@ pub(super) fn find_stackwalking_problem(
     }
 
     let mut modules_breakpad = result_breakpad.modules.clone().unwrap_or_default();
-    let mut modules_rust_minidump = result_rust_minidump.modules.clone().unwrap_or_default();
     fixup_modules(&mut modules_breakpad);
-    fixup_modules(&mut modules_rust_minidump);
+    let modules_rust_minidump = result_rust_minidump.modules.clone().unwrap_or_default();
```

Comment:
`modules_rust_minidump` will still need to be sorted by id though.

Reply:
oh, good catch.  I've opted to do this outside of the fixup function because I'd kid of like to see the cases where the modules are not transformed correctly already.